SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2307,25 +2293,24 @@ USpatialActorChannel* USpatialNetDriver::CreateSpatialActorChannel(AActor* Actor
 	check(NetConnection != nullptr);
 
 	USpatialActorChannel* Channel = nullptr;
-	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton))
+
+	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton) && !LoadBalanceStrategy->ShouldHaveAuthority(*Actor))
 	{
-		Channel = GlobalStateManager->AddSingleton(Actor);
+		return Channel;
+	}
```

Comment:
Currently, for complicated reasons, singletons initially have authority on every server initially. We need to explicitly prevent each server trying to creating an entity for them. Previously this was handled within GSM::AddSingleton (and based on GSM authority), now there's a static method in the Singleton component file (seemed best place vs add MORE to the netdriver)